eng/WpfArcadeSdk/tools/SdkReferences.targets

```diff
@@ -194,14 +194,14 @@
       
     It will get translated into something like this:
       <ItemGroup>
-        <Reference Include="C:\Users\username\.nuget\packages\microsoft.private.winforms\4.8.0-preview4.19170.4\lib\netcoreapp3.0\Accessibility.dll" />
+        <Reference Include="C:\Users\username\.nuget\packages\microsoft.private.winforms\4.8.0-preview4.19170.4\lib\netcoreapp3.0\Accessibility.dll"/>
       </ItemGroup>
    -->
   <Target Name="ResolveWinFormsReferences"
           Condition="'@(MicrosoftPrivateWinFormsReference)'!='' And $(TargetFramework.StartsWith('netcoreapp3'))">
     <ItemGroup>
-      <Reference Include="$(PkgMicrosoft_Private_Winforms)\ref\$(TargetFramework)\%(MicrosoftPrivateWinFormsReference.Identity).dll"
-                 Condition="Exists('$(PkgMicrosoft_Private_Winforms)\ref\$(TargetFramework)\%(MicrosoftPrivateWinFormsReference.Identity).dll')">
```

Comment:
Why?

Reply:
because the reference assembly can't be loaded at runtime so qualityvault (and any exe) would fail to run

Reply:
These are intended to be build time refs - not intended to supply runtime assemblies.

Reply:
I don't think conflating ref-assemblies with lib-assemblies is a good practice in general. We do that for WPF assemblies for very special reasons, but we don't have good reasons to do that for WinForms assemblies. 

These are used to build the product - and we should continue building strictly against the ref-assemblies only.

Reply:
As a general rule, any changes to build-orchestration that are needed by tests should not create side-effects upon product builds. 

We can think about better ways to achieve what you are trying to do here. For e.g., if the tests needs access to WinForms runtime-assemblies somehow, we'll need to solve that problem differently than to change what we build-against (ref-assemblies).

Reply:
sounds good